packages/cli/src/commands/accounts/index.ts

```diff
@@ -0,0 +1,24 @@
+import {Command} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import {current, list} from '../../lib/accounts/accounts'
+
+export default class AccountsIndex extends Command {
+  static description = 'list the Heroku accounts in your cache'
+
+  static example = 'heroku accounts'
+
+  async run() {
+    const accounts = list()
+    if (accounts.length === 0) {
+      ux.error('No accounts found in cache.')
```

Comment:
I noticed this was new error text (it's a very good sanity check, btw). And it looks like cx reviewed it, but we resolved the conversation without change. Was that intentional?

Reply:
Claire let me know that I'd missed this one, it's been updated. I must have resolved it by accident.